pallets/data-preservers/src/lib.rs

```diff
@@ -168,143 +318,53 @@ pub mod pallet {
 
         UnknownProfileId,
         NextProfileIdShouldBeAvailable,
+
+        /// Made for `AssignmentPayment` implementors to report a mismatch between
+        /// `ProviderRequest` and `AssignerParameter`.
+        AssignmentPaymentRequestParameterMismatch,
+
+        ProfileAlreadyAssigned,
+        ProfileNotAssigned,
+        ProfileIsNotElligibleForParaId,
+        WrongParaId,
+        MaxAssignmentsPerParaIdReached,
     }
 
     #[pallet::composite_enum]
     pub enum HoldReason {
         ProfileDeposit,
     }
 
+    #[deprecated]
```

Comment:
This makes CI error because of a deprecated warning... Can the storage item be removed now?

Reply:
It is used in the migration.

Reply:
We can make it not use it by quering directly by key, like here: https://github.com/moondance-labs/tanssi/blob/2fb1c0813d52c5c603a86ae46fec01545840e058/runtime/common/src/migrations.rs#L392

Reply:
In the migration we iterate over the keys, so it is more complex than accessing a single key.

Reply:
I believe we did this in some migration in the past

Reply:
yes here an example @nanocryk https://github.com/moonbeam-foundation/moonbeam/blob/44e47fd9e9fa462329743c07db7fda96840eb32c/pallets/asset-manager/src/migrations.rs#L54